src/components/Dialog/index.ts

```diff
@@ -1 +1,4 @@
 export {Dialog} from './Dialog.component';
+export type {DialogProps
```

Comment:
dialog props import the below types as well. Do we need to import them separately?

Reply:
Exported separately because `DialogProps` type can be either `DialogWithJsxAndOptionalTitle` or `DialogWithTitleAndOptionalJsx`. There might a case in plugin where you only want `DialogProps: DialogWithTitleAndOptionalJsx` so to add type there I export  them as well.